releases/release-6.1.4.md

```diff
@@ -0,0 +1,102 @@
+---
+title: TiDB 6.1.4 Release Notes
+---
+
+# TiDB 6.1.4 Release Notes
+
+发版日期：2023 年 2 月 8 日
+
+TiDB 版本：6.1.4
+
+试用链接：[快速体验](https://docs.pingcap.com/zh/tidb/v6.1/quick-start-with-tidb) | [生产部署](https://docs.pingcap.com/zh/tidb/v6.1/production-deployment-using-tiup) | [下载离线包](https://cn.pingcap.com/product-community/?version=v6.1.4#version-list)
+
+## 兼容性变更
+
+- Tools
+
+    - TiCDC
+
+        - 
+
+## 提升改进
+
+- PD
+
+    - 
+
+- TiFlash
+
+    - 降低了在高强度的更新负载下的 IOPS 和写放大 [#6460](https://github.com/pingcap/tiflash/issues/6460) @[flowbehappy](https://github.com/flowbehappy)
+
+- Tools
+
+    - TiCDC
+
+        - 
+    
+    - Lightning
+
+        - Change severity of clusterResourceCheckItem,emptyRegionCheckItem from Critical to Warn [#37654](https://github.com/pingcap/tidb/issues/37654) @[lance6716](https://github.com/lance6716) 
+
+## Bug 修复
+
++ TiDB
+
+    tidb
+
+    (dup: release-6.2.0.md > Bug fixes> TiDB)- Fix the issue that when you create a table, the default value and the type of a column are not consistent and are not automatically corrected [#34881](https://github.com/pingcap/tidb/issues/34881) @[Lloyd-Pottiger](https://github.com/Lloyd-Pottiger) @[mjonss](https://github.com/mjonss)
+
+    transaction
+
+    - session: fix data race in the LazyTxn.LockKeys [#40355](https://github.com/pingcap/tidb/issues/40355) @[HuSharp](https://github.com/HuSharp)
+    - Fix the case that INSERT/REPLACE might panic in a long session connection [#40351](https://github.com/pingcap/tidb/issues/40351) @[fanrenhoo](https://github.com/fanrenhoo)
+
```

Comment:
- Fix cursor read being canceled by GC [#39447](https://github.com/pingcap/tidb/issues/39447) @[zyguan](https://github.com/zyguan)
- Fix pessimistic autocommit configuration not working for point get plans [#39928](https://github.com/pingcap/tidb/issues/39928) @[zyguan](https://github.com/zyguan)